examples/library/Makefile

```diff
@@ -25,17 +25,36 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# This Makefile fragment defines all of the regression tests (and the
-# source path) for this sub-directory.
+# This (rather sparse) Makefile is responsible for building and
+# running regression tests on F1.  Run `make help` to see the
+# available targets. (The target `help` is defined in f1.mk)
+
+# environment.mk verifies the build environment and sets the following
+# makefile variables:
```

Comment:
This file is an amalgamation/move of the files tests.mk, and rules.mk and Makefile in regression/python/